execution/complete.go

```diff
@@ -4,27 +4,24 @@ import (
 	"time"
 
 	"github.com/mesg-foundation/core/pubsub"
-	"github.com/mesg-foundation/core/service"
 )
 
 // Complete marks an execution as complete and puts into the list of processed tasks.
 func (execution *Execution) Complete(outputKey string, outputData map[string]interface{}) error {
-	output, ok := execution.Service.Tasks[execution.Task].Outputs[outputKey]
-	if !ok {
-		return &service.TaskOutputNotFoundError{
-			TaskKey:       execution.Task,
-			TaskOutputKey: outputKey,
-			ServiceName:   execution.Service.Name,
-		}
+	var (
+		s       = execution.Service
+		taskKey = execution.Task
```

Comment:
Remove var section??
It is used only once in func

Reply:
i just like gofmt to align multiple vars together, more readable